Source/VCGeneration/ConditionGeneration.cs

```diff
@@ -881,8 +879,7 @@ private void AddDebugInfo(Cmd c, Dictionary<Variable, Expr> incarnationMap, List
           {
             foreach (var param in current.Params)
             {
-              if (param is IdentifierExpr identifierExpr)
-              {
+              if (param is IdentifierExpr identifierExpr) {
```

Comment:
Style consistency?

Reply:
What's your question?

Reply:
I see a curly brace put at the end of the line, but it seems that for loops in the same file have it on a second line.